### PR TITLE
Swissai model

### PR DESCRIPTION
**Pull request by haeggee**

### CODE REVIEW

**Comment by haeggee** [comment]
should we merge @AleHD ?

**Comment by AleHD** [comment]
My bad, sorry for the late reply. Yes, let's merge it, logits done with the megatron converters test suggest the implementation is correct, thanks! :D
